### ISSUE TITLE
Not getting proper convergence using the video classification tutorial.

### ISSUE THREAD

**Issue by lcskrishna** [facebookresearch/ClassyVision#623]
## 🐛 Bug
I have trained the resnext3d model using ucf101 from the video classification example of the tutorails. However, I am not able to see proper convergence of loss. I am originally experimenting this training on ROCm platform using AMD GPUs, however I am able to reproduce the same issue on CUDA platform as well. 
It would be great if someone can help us out what hyperparameters needs to be changed to get a proper convergence.

cc: @sunway513

## To Reproduce
1. Run the video classification example given in : [Video Classification Tutorial](https://github.com/facebookresearch/ClassyVision/blob/master/tutorials/video_classification.ipynb). 
2. After the task is created add the following line to use multiprocessing context as fork:
task.set_dataloader_mp_context("fork")
3. Update num_epochs to 90.

## Expected behavior
A proper convergence.

## Environment

 - PyTorch Version : 1.7.0a0+19fb0b8
 - OS (e.g., Linux): Linux 
 - How you installed PyTorch (`conda`, `pip`, source): source
 - Build command you used (if compiling from source):
 - Python version: python3.6
 - CUDA/cuDNN version: 11.0
 - ROCm version : rocm 3.7
 - GPU models and configuration: resnext3d.

**Comment by mannatsingh** [comment]
@lcskrishna the example in the UCF101 tutorial is a toy example, so I don't think it is expected to converge with those parameters. Also, if you use the LR setting in the tutorial and increase num_epochs to 90, the LR will still drop at epoch 1 and then stay there for the remaining 88 epochs as well. 

If you are testing out AMD GPUs, I would highly recommend first working with Image Classification - training a ResNet 50 on ImageNet. We have an example with a config which should work out of the box and the expected accuracy.

I personally haven't trained on UCF 101 myself, so I don't know the ideal hyper parameters to use, but it should be easy enough to figure out. @stephenyan1231 should have an idea. But again, I would recommend training on ImageNet using a ResNet 50 first since that is the most common use case and I will be able to help you out with that better.
